crates/criticaltrust/src/signatures/keychain.rs

```diff
@@ -158,6 +193,67 @@ mod tests {
         assert!(matches!(keychain.load(&other), Err(Error::UnsupportedKey)));
     }
 
+    impl Signable for String {
+        const SIGNED_BY_ROLE: KeyRole = KeyRole::Packages;
+    }
+
+    impl Signable for Vec<u8> {
+        const SIGNED_BY_ROLE: KeyRole = KeyRole::Revocation;
+    }
+
+    #[test]
+    fn test_load_all_revoked_content_empty() {
+        // Test `load_all` method with RevocationInfo but empty list.
+        let root = generate_key(KeyRole::Root);
+        let revocation = generate_trusted_key(KeyRole::Revocation, &root);
```

Comment:
(Not necessarily change request, just stylistic) Places like this are often an ideal place to use destructuring (eg `let (keypair, payload) = generate_trusted_key(...)`) as in a few months you may have forgotten what `.0` and `.1` are and need to check... then check again later in 4 months... then again...

Reply:
This was good! Thank you!